### ISSUE TITLE
Initialization with SVG?

### ISSUE THREAD

**Issue by LizardBanks** [go-echarts/go-echarts#165]
https://github.com/go-echarts/go-echarts/blob/fa2cd12fc154af880978b041f5519a9476cd0ba6/opts/global.go#L20 - 
Documentation suggests SVG is supported but initialization implies only canvas support works..
Any idea if it's possible to download the map as an SVG using the Toolbox feature?

**Comment by syrm** [comment]
Same question, is it possible to render into SVG ?

**Comment by pablojimpas** [comment]
It would be really nice to have support for SVG rendering, potentially enabling [SSR with hydration](https://echarts.apache.org/handbook/en/how-to/cross-platform/server/) as well. You could send just an SVG to clients that don't support JavaScript (or if you want to show a preview quickly or save bandwidth/resources). Then, if the client supports JavaScript, you will replace that element with the canvas rendered version of the same chart once the library is ready.

**Comment by mitar** [comment]
I made PR https://github.com/go-echarts/go-echarts/pull/334.
